contracts/mocks/MockBlockRelay.sol

```diff
@@ -228,4 +241,28 @@ contract MockBlockRelay is BlockRelayInterface {
     }
     return _root == tree;
   }
+
+  /// @dev This function checks if the relayer has been paid
+  /// @param _blockHash Hash of the block header
+  /// @return true if the relayer has been paid, false otherwise
+  function isRelayerPaid(uint256 _blockHash) public view override returns(bool){
+    return blocks[_blockHash].isPaid;
+  }
+
+  /// @dev Pay the block reward to the relayer in case it has not been paid before
+  /// @param _blockHash Hash of the block header
```

Comment:
```suggestion
  /// @dev Pay the block reward to the relayer in case it has not been paid before.
  /// @param _blockHash Hash of the block header.
```